brigade-worker/package.json

```diff
@@ -38,7 +38,7 @@
     "typescript": "^3.2.2"
   },
   "dependencies": {
-    "@brigadecore/brigadier": "^0.5.0",
+    "@brigadecore/brigadier": "radu-matei/brigadier#job-volume-config-out",
```

Comment:
After https://github.com/brigadecore/brigadier/pull/22 is merged and a new version is published, update this.